terraform/environments/nomis/ssm-documents/run-ansible-patches.yaml

```diff
@@ -0,0 +1,136 @@
+---
+schemaVersion: "2.2"
+description: "run modernisation-platform-configuration-management ansible-playbooks against EC2 instances"
+parameters:
+  PlaybookFile:
+    type: "String"
+    description: "(Optional) The Playbook file to run (including relative path). If the main Playbook file is located in the ./automation directory, then specify automation/playbook.yml."
+    default: "site.yml"
+  ExtraVariables:
+    type: "String"
+    description: "(Optional) Additional variables to pass to Ansible at runtime. Enter key/value pairs separated by a space. For example: color=red flavor=cherry"
+    default: "SSM=True"
+    displayType: "textarea"
+    allowedPattern: "^$|^\\w+\\=(([^\\s|:();&]+)|('[^|:();&]+'))(\\s+\\w+\\=(([^\\s|:();&]+)|('[^|:();&]+')))*$"
+  Check:
+    type: "String"
+    description: "(Optional) Use this parameter to run a check of the Ansible execution. The system doesn’t make any changes to your systems. Instead, any module that supports check mode reports the changes it would make rather than making them. Modules that don’t support check mode take no action and don’t report changes that would be made."
+    allowedValues:
+      - "True"
+      - "False"
+    default: "True"
+  Verbose:
+    type: "String"
+    description: "(Optional) Set the verbosity level for logging Playbook executions. Specify -v for low verbosity, -vv or –vvv for medium verbosity, and -vvvv for debug level."
+    allowedValues:
+      - "-v"
+      - "-vv"
+      - "-vvv"
+      - "-vvvv"
+    default: "-v"
+  TimeoutSeconds:
+    type: "String"
+    description: "(Optional) The time in seconds for a command to be completed before it is considered to have failed."
+    default: "3600"
+  Role:
+    type: "String"
+    description: "(Optional) The role to run, not that not all roles are applicable for all targets!"
+    allowedValues:
+      - "all"
```

Comment:
Can we just allow any role here for the sake of simplicity, if the user gets the role wrong, nothing will get run anyway